classes/simplemap.php

```diff
@@ -2289,27 +2289,27 @@ function show_toolbar( $title = '' ) {
 			<table class="sm-toolbar" cellspacing="0" cellpadding="0" border="0">
 				<tr>
 					<td class="sm-page-title">
-						<h2><?php _e( $title, 'simplemap' ); ?></h2>
+						<h2><?php _e( $title, 'SimpleMap' ); ?></h2>
 					</td>
 					<td class="sm-toolbar-item">
 						<a href="http://simplemap-plugin.com" target="_blank"
-						   title="<?php _e( 'Go to our website', 'simplemap' ); ?>">SimpleMap <?php _e( 'Home Page', 'simplemap' ); ?></a>
+						   title="<?php _e( 'Go to our website', 'SimpleMap' ); ?>">SimpleMap <?php _e( 'Home Page', 'SimpleMap' ); ?></a>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 138 characters

<!-- Comment by [SideCI](https://sideci.com) -->